python_modules/dagster/dagster_tests/daemon_sensor_tests/test_sensor_run.py

```diff
@@ -223,6 +223,18 @@ def run_cursor_sensor(context):
     return RunRequest(run_key=None, run_config={}, tags={})
 
 
+@sensor(job_name="the_job")
+def start_skip_sensor(context):
+    # skips the first tick after a start
+    if (
+        not context.last_completion_time
+        or context.last_start_time
+        and context.last_start_time > context.last_completion_time
```

Comment:
ok i think i understand the motivation for the boolean now :) Making the change to last_start_time (and maybe renaming last_completion_time to last_tick_completion_time?) will help make this clearer i think